CHANGELOG.md

```diff
@@ -80,6 +80,8 @@ Breaking changes:
     * Remove `IUnknownPropertiesConsumer`
 * Bump: Kotlin to 1.5 and compatibility to 1.4 for sentry-android-timber (#1815)
 
+* Feat: Allow setting SDK info (name & version) in manifest (#2016)
```

Comment:
Move it under `unreleased`.

Reply:
This is duplicated now, changelog entry is in both places

Reply:
sorry about the mess caused by the conflicts while rebasing. removed the latter entry.